lib/brains/models.py

```diff
@@ -20,9 +21,35 @@ class Meta(Model.Meta):
         db_table = 'buyer_braintree'
 
     def get_uri(self):
-        return reverse('braintree:buyer', kwargs={'pk': self.pk})
+        return reverse('braintree:mozilla:buyer', kwargs={'pk': self.pk})
 
     def save(self, *args, **kw):
         if not self.pk:
             self.braintree_id = str(self.buyer.pk)
         return super(BraintreeBuyer, self).save(*args, **kw)
+
+
+class BraintreePaymentMethod(Model):
+
+    """A holder for braintree specific payment method."""
+
+    active = models.BooleanField(default=True)
+    braintree_buyer = models.ForeignKey(BraintreeBuyer)
+    # An id specific to the provider.
+    provider_id = models.CharField(max_length=255)
+    # The type of payment method eg: card, paypal or bitcon
+    type = models.PositiveIntegerField(choices=(
+        (PAYMENT_METHOD_CARD, PAYMENT_METHOD_CARD),
+    ))
+    # Details about the type, eg Amex, Orange.
+    type_name = models.CharField(max_length=255)
+    # For credit cards, this is the last 4 numbers, could be a truncated
+    # phone number or paypal account for example.
+    truncated_id = models.CharField(max_length=255)
```

Comment:
this doesn't define the ``counter`` column. Do we need that column? If so, could you define it and add a comment about it what it means?